frontend/src/utils.js

```diff
@@ -10,3 +12,18 @@ export const postData = async (url, data) => {
 
   return response.ok;
 };
+
+export const useQuery = () => {
+  const queryString = useLocation().search;
+  return new URLSearchParams(queryString);
+};
+
+export const isAuthenticated = () => {
+  const token = window.localStorage.getItem("til_token");
+  if (token) return true;
+  else return false;
```

Comment:
`else` not needed
```suggestion
  if (token) return true;
  
  return false;
```

Reply:
removed else statement and changed to proposed